src/vaadin-number-field.html

```diff
@@ -151,9 +160,9 @@
         __add(sign) {
           const incr = sign * (this.step || 1);
           // Behave like native number input adjusting to the next exact multiple of step.
-          this.value = (incr + (incr * Math.floor((parseFloat(this.value || 0) / incr).toFixed(1)))).toFixed(this.__decimals);
+          this.value = this.focusElement.value =
+            (incr + (incr * Math.floor((parseFloat(this.value || 0) / incr).toFixed(1)))).toFixed(this.__decimals);
           this.dispatchEvent(new CustomEvent('change', {bubbles: true}));
-          this.focusElement.focus();
```

Comment:
This didn't work however, the field still gets focused on first time the button is clicked so I get the virtual keyboard (on both iOS and Android) which goes away the second time the button is clicked